src/Page/Guides/View.elm

```diff
@@ -37,16 +37,17 @@ view model =
 
                     Success list ->
                         List.concat [ Page.Guides.Data.teaserListFromGuideDict model.language model.content.guides, list ]
+
+        headerText : String
+        headerText =
+            if String.length model.query > 0 then
+                t <| GuidesTitleFiltered (String.fromInt <| List.length model.search) model.query
```

Comment:
We aren't using this sytax elsewhere. It's probably best for readabilitty if we stick with () for this direction of application when there are one or 2 functions to apply and |> where there is a longer chain of functions.